sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -117,14 +128,42 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
       case NonFatal(_) => None
     }
     val isTemp = sessionCatalog.isTempView(tableIdent)
+    val qualifier =
+      Array(metadata.map(_.identifier.database).getOrElse(tableIdent.database).orNull)
```

Comment:
shall we include catalog name (`spark_catalog`) as well?

Reply:
This is `makeTable(tableIdent: TableIdentifier)`, and the `TableIdentifier` contains no catalog name. Are you suggesting that I use `getCurrrentCatalog` (not exists but we can add one) to get the catalog name?

Reply:
Will use `:String` to solve this issue.